install/util/wait.go

```diff
@@ -24,3 +29,17 @@ func WaitFor(interval, timeout time.Duration, isReady func() (bool, error)) erro
 		}
 	}
 }
+
+func WithDefaultRetry(invocation func() (interface{}, error)) (interface{}, error) {
+	return withRetry(defaultRetryNo, defaultSleepDuration, invocation)
+}
+
+func withRetry(count int, sleep time.Duration, invocation func() (interface{}, error)) (interface{}, error) {
```

Comment:
This should work, however you can also consider using some go libraries for retry. 
`"github.com/avast/retry-go"` 

Usage example in provosioner:

`	err = retry.Do(func() error {
		_, err := g.shootClient.Update(context.Background(), shoot, v1.UpdateOptions{})
		return err
	}, retry.Attempts(5))`

Reply:
I wanted to keep our implementation minimal. Would mind leaving the given code as it is right now without introducing another dependency to the module.